pages/instructorAssessmentManualGrading/assessment/assessment.sql

```diff
@@ -10,13 +21,10 @@ WITH instance_questions_with_submission AS (
         JSONB_AGG(DISTINCT jsonb_build_object('user_id', lgu.user_id, 'name', lgu.name, 'uid', lgu.uid)) FILTER (WHERE iq.last_grader IS NOT NULL) AS actual_graders
     FROM
         assessment_questions aq
-        JOIN instance_questions iq ON (iq.assessment_question_id = aq.id)
+        JOIN selected_instance_questions iq ON (iq.assessment_question_id = aq.id)
         LEFT JOIN users agu ON (agu.user_id = iq.assigned_grader)
         LEFT JOIN users lgu ON (lgu.user_id = iq.last_grader)
     WHERE aq.assessment_id = $assessment_id
```

Comment:
```suggestion
        JOIN instance_questions iq ON (iq.assessment_question_id = aq.id)
        LEFT JOIN users agu ON (agu.user_id = iq.assigned_grader)
        LEFT JOIN users lgu ON (lgu.user_id = iq.last_grader)
    WHERE aq.assessment_id = $assessment_id
      AND iq.status != 'unanswered'
```
and get rid of the `selected_instance_questions` subquery altogether.

Reply:
Excellent point! Done in 635fbae596a69024ab112ff4f9468eb2dd241f88